packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -4,28 +4,70 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { Aliases, AuthInfo } from '@salesforce/core';
-import { ForceConfigGet } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
-
+import {
+  Aliases,
+  AuthInfo,
+  ConfigAggregator,
+  ConfigFile
+} from '@salesforce/core';
+import * as vscode from 'vscode';
 export class OrgAuthInfo {
   public static async getDefaultUsernameOrAlias(
     vscodePath: string
   ): Promise<string | undefined> {
-    const forceConfig = await new ForceConfigGet().getConfig(
+    if (
+      vscode.workspace &&
+      vscode.workspace.workspaceFolders &&
+      vscode.workspace.workspaceFolders[0]
+    ) {
+      const rootPath = vscode.workspace.workspaceFolders[0].uri.fsPath + '/';
```

Comment:
You can replace this with `getRootWorkspacePath()`. 

I'm not sure building the path like this and adding `.sfdx/` on line 26 will work on windows. Can we replace it by building the uri using `path.join()` ?

Reply:
Yeah, that's a good point. I'll use `getRootWorkspacePath()` instead and `path.join()` where needed